tests/integration/security/testdata/authz/v1beta1-conditions.yaml.tmpl

```diff
@@ -96,14 +96,14 @@ metadata:
 spec:
   selector:
     matchLabels:
-      app: c
+      app: {{ .cSet }}
   rules:
   - to:
     - operation:
-        paths: ["/source-ip-notValues-b"]
+        paths: ["/source-ip-notValues-{{ .b }}"]
     when:
     - key: source.ip
-      notValues: ["{{ .IpB }}"]
+      notValues: ["{{ .ipB }}"]
```

Comment:
Pretty much the same comment as above for all of the template vars. 

It's probably outside the scope of this PR, but I think adding templating now just increases the complexity without adding value. 

Eventually, we should probably look at something like the networking tests have, where the config lives on the test case:

https://github.com/istio/istio/blob/da6178604559bdf2c707a57f452d16bee0de90c8/tests/integration/pilot/common/traffic.go#L44-L47

And use a somewhat standard set of template vars and allowing custom template vars for individual tests that need them:

https://github.com/istio/istio/blob/da6178604559bdf2c707a57f452d16bee0de90c8/tests/integration/pilot/common/traffic.go#L93-L105


Ideally, we merge the security context stuff, the traffictestcase and rbacutil, but also seems like a bigger effort. 


In short – it's probably better not to add templating now, it seems like it would add more work to refactor and use `echotest`. I don't feel strongly though.

Reply:
I had discussed with @yangminzhu , this is because the security tests require multiple workloads as destination. e.g. some tests apply multiple security policies to multiple workloads, and expect each workload to behave differently. e.g. a workload `a` will receive different response code when sending requests to `b`, `c`, and `d`. There are tests which verify that different source workloads sending request to the same destination will receive different response code. 

Originally I tried to split the policy to small pieces, and apply to the single destination workload (pod/vm), the feedback from the review is that this approach reduces the test coverage. So I reverted that change back and keep using workloads `a`, `b`, `c`... etc. Once we have support for multiple VMs, I can add a group of VMs to rerun the tests in one more round (e.g. pass vm a, vm b, vm c to those `a`, `b`, `c`...).